docs/en/administration/data_migration_tool.md

```diff
@@ -147,16 +184,19 @@ The description of the parameters is as follows:
 | target_fe_query_port                      | The query port (`query_port`) of the target cluster's FE.    |
 | target_cluster_user                       | The username used to log in to the target cluster. This user must be granted the OPERATE privilege on the SYSTEM level. |
 | target_cluster_password                   | The user password used to log in to the target cluster.      |
+| include_data_list                         | The databases and tables that need to be migrated, with multiple objects separated by commas (`,`). For example: `db1, db2.tbl2, db3`. This item takes effect prior to `exclude_data_list`. If you want to migrate all databases and tables in the cluster, you do not need to configure this item. |
+| exclude_data_list                         | The databases and tables that do not need to be migrated, with multiple objects separated by commas (`,`). For example: `db1, db2.tbl2, db3`. `include_data_list` takes effect prior to this item. If you want to migrate all databases and tables in the cluster, you do not need to configure this item. |
+| target_cluster_storage_volume             | The storage volume used to store tables in the target cluster when the target cluster is a shared-data cluster. If you want to use the default storage volume, you do not need to specify this item. |
+| target_cluster_replication_num            | The replication number of the tables to be created in the target cluster. If you want to use the same replication number as the source cluster, you do not need to specify this item. |
 | meta_job_interval_seconds                 | The interval, in seconds, at which the migration tool retrieves metadata from the source and target clusters. You can use the default value for this item. |
+| meta_job_threads                          | The number of threads used by the migration tool to obtain metadata from the source and the target cluster. You can use the default value for this item. |
```

Comment:
obtain metadata from the source and the target cluster.  --- obtain metadata from the source and target clusters.